docs/plugins/development/index.md

```diff
@@ -138,29 +138,41 @@ Any additional apps must be installed within the same Python environment as NetB
 
 ## Create setup.py
 
-`setup.py` is the [setup script](https://docs.python.org/3.8/distutils/setupscript.html) used to package and install our plugin once it's finished. The primary function of this script is to call the setuptools library's `setup()` function to create a Python distribution package. We can pass a number of keyword arguments to control the package creation as well as to provide metadata about the plugin. An example `setup.py` is below:
+`pyproject.toml` is the [configuration file](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/) used to package and install our plugin once it's finished. The primary function of this script is to call the build system to create a Python distribution package. We can pass a number of keyword arguments to control the package creation as well as to provide metadata about the plugin. An example `pyproject.toml` is below:
```

Comment:
> The primary function of this script

Maybe a bid pedantic, but this is no longer a script. We should more accurately explain how the `.toml` file is processed.